hat/assets/js/apps/Iaso/domains/assignments/hooks/requests/useSaveAssignment.ts

```diff
@@ -16,13 +16,14 @@ export const saveAssignment = async (
     return postRequest(endpoint, body);
 };
 
-export const useSaveAssignment = (
-    callback: () => void = () => null,
-): UseMutationResult => {
-    const onSuccess = () => callback();
+export const useSaveAssignment = (): UseMutationResult => {
+    const queryClient = useQueryClient();
+    const onSuccess = () => {
+        queryClient.invalidateQueries('orgUnits');
+        queryClient.invalidateQueries('assignmentsList');
```

Comment:
workaround to invalidate multiple query keys?

Reply:
yes, could not find another way